### ISSUE TITLE
Adding new folders

### ISSUE THREAD

**Issue by bil-elmoussaoui** [numixproject/numix-folders#78]
Hello , 
I just wanted to ask if you are going to add some new folders? 
- [ ] Steam folder
- [ ] VirtualBox/Vmware/Gnome Box folder 
- [ ] Books folder (alreay reported in Numix base)
- [ ] Movies/Series folder 
- [ ] Github folder
- [ ] Backup - might not be really needed -

Thank you for your awesome work !! and i guess this will not be as hard to do as before.

**Comment by wa4557** [comment]
The first two are already there!

**Comment by bil-elmoussaoui** [comment]
@wa4557 i found only Steam folder, can you please give me the filename of the second one?

**Comment by wa4557** [comment]
`*-folder-vm.svg`

**Comment by bil-elmoussaoui** [comment]
Thank you ;)

**Comment by Foggalong** [comment]
GitHub is a wontfix because it's a bit too specific - we do have Projects and System folders though :) the other are doable though won't be a priority to fulfil. but same as always we'll accept community contributions for any of them!

**Comment by wa4557** [comment]
I still want to have and do a books folder. But its a tedious work :(

**Comment by wa4557** [comment]
I made book folders cf#79. Since this a hell lot of work I won't make any more :( so if anyone wants to make the movie/series folder fine but since we already have a video folder I suppose this might as well be a wontfix

**Comment by bil-elmoussaoui** [comment]
Well, thank you very much for your hardwork :smile_cat:  maybe somebody from the community will want to make the others.

**Comment by palob** [comment]
Now with the custom colour switching implemented in the script are the coloured svgs still needed?
Couldn't we have the predefined colours defined in a ``csv`` and have the script create the svgs?

**Comment by Foggalong** [comment]
The coloured SVGs are needed for Folder-Color support, so no unfortunately :(

**Comment by palob** [comment]
Ah OK, but at least when doing new folders the coloured ones could be created using the script.

**Comment by Foggalong** [comment]
It gets a bit complicated if you want to generate some icons from one colour but not others. You could use the script to generate them in the first place but they'd need to stay in the theme afterward

**Comment by palob** [comment]
Yep, that's what I meant.

**Comment by Foggalong** [comment]
ooop! sorry, I gotya :P

**Comment by bil-elmoussaoui** [comment]
I added Recording folder! if you're a student like me, i do use  the gnome recording application to record some courses! helps me a lot

**Comment by MrTimscampi** [comment]
Would it be possible to add a GOG Games folder ?

It could be a simple folder with the GOG.com logo on it, much like the Steam folder.

**Comment by Foggalong** [comment]
Yeah, but same as for the others only by user contribution. Too niche to cover otherwise

**Comment by Jacajack** [comment]
Hi everyone! 
I recently came up with an idea, that I'd be nice to have some folders with mime-type based symbols. For example one for C code, and one for JavaScript, and so on...

This is what I mean:
![selection_815](https://cloud.githubusercontent.com/assets/6173262/17891818/6a851bb0-693e-11e6-927a-467e4bddbc39.png)
*I made this yesterday, in like 2 minutes, without any guideline, that's why it's ugly*

I can create and contribute some of those, if you think the idea makes sense.

Also, what about colors? Is there some script that generates colorful icons available, or should I create mine? (I found some `custom-folder-*.svg` icons, I assume those are the base ones)

**Comment by wa4557** [comment]
I guess new folders are ok (and welcome probably) :) Be warned though this is a LOT of work (I actually started to add a new folder and gave up halfway through it. The problem is, that in order to get them merged you need to provide a lot of different sizes (16px, 22px, 24px, 32px, 48px, 64px) and 6 different styles makes a total of 36 distinct icons each aligned to pixel grid. 

Then you also need a lot of different colors (this can be scripted though)

So yeah, new folders are cool, but a lot of work

**Comment by Jacajack** [comment]
Oh, I didn't realize it's that much work...

Maybe it would be possible to rescale icons with some other script, then only align the symbols in Inkscape? And I believe script for copying symbols over different styles would be useful.

I can try to do this, if that way of developing icons is acceptable ;)

**Comment by palob** [comment]
You don't need a script for resizing the icons, the folders itself are already present for all sizes :)

**Comment by Jacajack** [comment]
I meant to say, script for rescaling the new symbols themselves :)

**Comment by Jacajack** [comment]
Ok, so I made a prototype...

This is how it works:
 - Create single 16x16 symbol, and then optimize it (I used `svgo` because `svgclean` was removing my titled group, but that's something I'm going to work on)
The only requirement is that symbol must be in group with title 'symbol'.
![selection_825](https://cloud.githubusercontent.com/assets/6173262/17925090/cc33cfde-69eb-11e6-97da-11ebb1041948.png)
*This is the one I used for testing*


 - Run first script to multiply symbol over SVG files of different sizes
*Like this:*
 ![selection_826](https://cloud.githubusercontent.com/assets/6173262/17925176/23f9043c-69ec-11e6-8b89-8b8cd9f7be3a.png)
*And you get this:*
![image](https://cloud.githubusercontent.com/assets/6173262/17925196/3c0a826c-69ec-11e6-924d-15f27608b27e.png)

 - Manually rescale and fit symbols somewhere in the middle in the each file.
 - Run `./paste.pl` and it will merge symbols with base folder icons according to `paste.json`, which looks like this currently:
```json
[
    ["base/48/custom-folder.svg", "symbols/48.svg", "icons/48/custom-folder-test.svg"]
]
```
- This is result:
![selection_828](https://cloud.githubusercontent.com/assets/6173262/17925504/87f0bf1a-69ed-11e6-8662-af4069a35c52.png)

- Then there are only minor edits left to do, depending on icon size and style

What do you think about this?

Here's code: https://gist.github.com/Jacajack/f1f065c49ce3cae9cf6cc9d85426c9fe
*By the way, I'd like to officially announce that I'm going to hell, because I'm parsing SVG files with regular expressions... So, if are Perl programmer, do not go there.*

**Comment by Foggalong** [comment]
If you can get it to work reliably so that the quality is still there, I'd be happy to merge additional folder designs into the repo.

**Comment by Foggalong** [comment]
Added Android into the list following the closure of #193
